CONTRIBUTING.md

```diff
@@ -104,6 +107,22 @@ To pick up an issue:
 
 ## Additional Developer Notes
 
+### Add a new CLI command
+- Create a handler implementation under `handler/*` directory
+- Make proper godoc comments for the implementation
+- Add the handler function to the interface under `handler/*/interface.go`
+- Update the MockHandler under `handler/*/interface.go`
+- Create unit tests for this handler implementation
+- Create a function to document the swagger API `func _(){}`
+- Create corresponding API route handler in `api/api.go`
+- Create a CLI command under `cmd/*` directory
+- Update `cmd/app.go` to register the new command
+- Add the CLI command unit test in one of `cmd/*_test.go`
+- Optionally add CLI command integration tests in `cmd/functional_test.go`
+- Run `make generate` or `go generate ./...`
+- Add the swagger API unit test in `api/api_test.go`
+- Optionally add swagger API integration tests in `cmd/api_test.go`
```

Comment:
@hannahhoward @elijaharita 
See if above makes sense to you

Reply:
i think it makes sense!